v2/firmware/src/effect_hsi_control.cpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * Effect: Controllable HSI values
+ *
+ * Copyright (c) 2014--2015 Coredump Rapperswil
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the “Software”), to
+ * deal in the Software without restriction, including without limitation the
+ * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+ * sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ * IN THE SOFTWARE.
+ */
+
+// Get arduino headers
+#include <Arduino.h>
+
+// Get pin definitions
+#include "pins.h"
+#include "hsi.h"
+
+// Main loop
+void effect_hsi_control(void) {
+
+  float val_h = (analogRead(POT_1) * 360.0f) / 1023.0f;
+  float val_s = 1.0f-analogRead(POT_2) / 1023.0f;
+  float val_i = 1.0f-analogRead(POT_3) / 1023.0f;
```

Comment:
Maybe add some comments about the range mapping of the values. Something like
``// map POT_1 value to 0..360°``